src/frontend/komponenter/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -100,8 +100,9 @@ const Behandlinger: React.FC<IBehandlingshistorikkProps> = ({ minimalFagsak }) =
                                     new Date(a.behandlingEllerTilbakekreving.opprettetTidspunkt)
                                 )
                             )
-                            .map((behandling: BehandlingTabellobjekt) => (
+                            .map((behandling: BehandlingTabellobjekt, index) => (
                                 <Behandling
+                                    key={index}
```

Comment:
Key bør helst ikke være index, dersom indeksposisjonen kan endre seg. Det kan skape trøbbel for React. Vurder å bruke en annen verdi, f.eks. behandlingsid

Reply:
Endrer til behandlingsid 👍